internal/coremsgs/en_config_descriptions.go

```diff
@@ -20,301 +20,265 @@ import (
-	ConfigSharedstorageIpfsGatewayMaxIdleConns          = ffm("config.sharedstorage.ipfs.gateway.maxIdleConns", "TBD")
-	ConfigSharedstorageIpfsGatewayURL                   = ffm("config.sharedstorage.ipfs.gateway.url", "TBD")
-
-	ConfigSharedstorageIpfsGatewayProxyURL = ffm("config.sharedstorage.ipfs.gateway.proxy.url", "TBD")
-
-	ConfigSubscriptionMax = ffm("config.subscription.max", "TBD")
+	ConfigPrivatemessagingBatchAgentTimeout = ffc("config.privatemessaging.batch.agentTimeout", "How long to keep around a batching agent for a sending identity before disposal", timeDurationType)
+	ConfigPrivatemessagingBatchPayloadLimit = ffc("config.privatemessaging.batch.payloadLimit", "The maximum payload size of a private message data exchange payload", byteSizeType)
+	ConfigPrivatemessagingBatchSize         = ffc("config.privatemessaging.batch.size", "The maximum number of messages in a batch for private messages", intType)
+	ConfigPrivatemessagingBatchTimeout      = ffc("config.privatemessaging.batch.timeout", "The timeout to wait for a batch to fill, before sending", timeDurationType)
 
-	ConfigSubscriptionDefaultsBatchSize = ffm("config.subscription.defaults.batchSize", "TBD")
-
-	ConfigTokensConnector             = ffm("config.tokens[].connector", "TBD")
-	ConfigTokensCustomClient          = ffm("config.tokens[].customClient", "TBD")
-	ConfigTokensExpectContinueTimeout = ffm("config.tokens[].expectContinueTimeout", "TBD")
-	ConfigTokensHeaders               = ffm("config.tokens[].headers", "TBD")
-	ConfigTokensIdleTimeout           = ffm("config.tokens[].idleTimeout", "TBD")
-	ConfigTokensMaxIdleConns          = ffm("config.tokens[].maxIdleConns", "TBD")
-	ConfigTokensName                  = ffm("config.tokens[].name", "TBD")
-	ConfigTokensPlugin                = ffm("config.tokens[].plugin", "TBD")
-	ConfigTokensURL                   = ffm("config.tokens[].url", "TBD")
-
-	ConfigTokensProxyURL = ffm("config.tokens[].proxy.url", "TBD")
-
-	ConfigUIEnabled = ffm("config.ui.enabled", "Enables the web user interface")
-	ConfigUIPath    = ffm("config.ui.path", "The file system path which contains the static HTML, CSS, and JavaScript files for the user interface")
+	ConfigPublicstorageType = ffc("config.publicstorage.type", "**Deprecated** See `config.sharedstorage.type` instead.", stringType)
+
+	ConfigPublicstorageIpfsAPIURL          = ffc("config.publicstorage.ipfs.api.url", "The URL for the IPFS API", "URL "+stringType)
+	ConfigPublicstorageIpfsAPIProxyURL     = ffc("config.publicstorage.ipfs.api.proxy.url", "The URL for the IPFS proxy", "URL "+stringType)
+	ConfigPublicstorageIpfsGatewayURL      = ffc("config.publicstorage.ipfs.gateway.url", "The URL for the IPFS Gateway", "URL "+stringType)
+	ConfigPublicstorageIpfsGatewayProxyURL = ffc("config.publicstorage.ipfs.gateway.proxy.url", "The URL for the IPFS Gateway proxy", "URL "+stringType)
```

Comment:
This might applied to other similar `.proxy` entires, but particularly for IPFS. This sounds like it's mandatory, but actually it's more like `Optional HTTP proxy server to use when connecting to the IPFS Gateway`